### PR TITLE
add xiaokang_zhang info

### PR DESCRIPTION
**Pull request by zxkjack123**
- Add Xiaokang Zhang's information in source/people/main.rst
- Add a profile of Xiaokang Zhang. source/people/xzhang.rst
- Add a picture of Xiaokang Zhang. source/people/xiaokang_zhang.jpg

### CODE REVIEW

File: source/people/xzhang.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+| Engineering Research Building
+| 1500 Engineering Drive
+| Madison, WI 53703
+| xzhang965@wisc.edu
+| Phone (608) 515-2262
+
+Graduate Appointment
+--------------------
+
+I am a joint PdD. student currently working on sub-voxel activion in shutdown dose rate calculations of fusion energy systems.
```
Comment (pshriwise) [inline source/people/xzhang.rst]: type here PpD. -> PhD

Comment (pshriwise) [reply source/people/xzhang.rst]: The description of your project is very good :)

File: source/people/xzhang.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+| Engineering Research Building
+| 1500 Engineering Drive
+| Madison, WI 53703
+| xzhang965@wisc.edu
+| Phone (608) 515-2262
+
+Graduate Appointment
+--------------------
+
+I am a joint PhD. student currently working on sub-voxel activion in shutdown dose rate calculations of fusion energy systems.
```
Comment (gonuke) [inline source/people/xzhang.rst]: "activion" -> "activation"

**Comment by pshriwise** [comment]
This looks good to me @zxkjack123. Thanks! 

I'll let @gonuke merge once he's had a chance to look over your changes as well.

**Comment by gonuke** [comment]
Looks good to me..  - merging and publishing.  Thanks @zxkjack123

**Pull request merged**